### DISCUSSION TITLE
TabMail of ribbon.xml not found.

### DISCUSSION THREAD

**Discussion by ca-roshan** [0dteam/Phishing-Reporter#12] [Q&A]
Hello, I have tried your solution, it works great. But for some clients it gives an error,
![image001](https://github.com/0dteam/Phishing-Reporter/assets/132203007/129cd799-c54d-4753-955f-681953dc1649)

Have any one faced this problem, any way to handle this?

**Comment by 0dteam** [comment]
Hello. I am glad you like the solution.
For the issue you have provided, here is a solution:
https://stackoverflow.com/a/37159661

Or you may uncheck "Show add-in user interface errors" option (in Options -> Advanced)
![image](https://github.com/0dteam/Phishing-Reporter/assets/56728081/9c306107-49a4-4801-baec-702d1debf00f)
